og_context/og_context.module

```diff
@@ -199,27 +199,44 @@ function og_context($group_type = 'node', $group = NULL, $account = NULL, $check
   global $user;
   // User account is sent.
   $account = $account ? $account : user_load($user->uid);
-  if (!$check_access) {
-    // Check if we already have context only if we don't have to check user's
-    // permissions to access the current group.
-    $context = &drupal_static(__FUNCTION__, FALSE);
-    if (empty($group) && $context !== FALSE) {
-      // Determine if we can return cached values.
-      if (empty($context)) {
-        // We already tried to find a context, but couldn't.
-        return FALSE;
-      }
 
-      if ($context['group_type'] == $group_type) {
-        // Return the cached values.
-        return $context;
-      }
+  $id = FALSE;
+  if ($group) {
+    list($id) = entity_extract_ids($group_type, $group);
+  }
+
+  $cache_keys = array(
+    $group_type,
+    $id,
```

Comment:
This right here is the cause of not being able to retrieve the [current active groups](https://www.drupal.org/node/2720337). It makes $id required from $group, even if we just want a list of the active groups.